docs/build-windows.md

```diff
@@ -2,16 +2,25 @@
 
 Windows builds can be created using:
 
-- MSVC compiler, Visual Studio 2022 IDE suite, and vcpkg to provide dependencies
-  *(recommended)*.
+- MSClang compiler, Visual Studio 2022 IDE suite, and vcpkg to provide
```

Comment:
Pedantic comment: I tried searching for "MSClang" but it doesn't seem to be a thing. "Visual Studio with Clang/LLVM" seems to be how people call this, including Microsoft.

So I'm suggesting this instead: 

> Visual Studio 2022 IDE suite with Clang/LLVM compiler, and vcpkg to provide dependencies.